steps/container.py

```diff
@@ -35,7 +35,8 @@
 # A future version of Cekit will expose this to us, for now we hard-code
 DOCKER_API_VERSION = "1.35"
 
-d = docker.APIClient(version=DOCKER_API_VERSION)
+base_url = os.environ.get("CTF_API_SOCK", 'unix://var/run/docker.sock')
```

Comment:
What is the reasoning for the name `CTF_API_SOCK`?

Reply:
It's following the convention of the existing (undocumented) configuration environment variables, CTF_DOCKER_VOLUMES and CTF_DOCKER_ENV. (The third I am aware of, BEHAVE_TIMEOUT, doesn't follow that convention: but you renamed it from CTF_WAIT_TIME in #45).

I think CTF_ refers to "container testing framework" which might have been a name we used before cekit, I can't really remember. I'm not fond of it.

Reply:
I agree, I'm not fond of `CTF_`. I would be happy to rename them (and document them!).